components/console/introduction.rst

```diff
@@ -409,8 +409,8 @@ tools capable of helping you with different tasks:
 
 * :doc:`/components/console/helpers/questionhelper`: interactively ask the user for information
 * :doc:`/components/console/helpers/formatterhelper`: customize the output colorization
-* :doc:`/components/console/helpers/progresshelper`: shows a progress bar
-* :doc:`/components/console/helpers/tablehelper`: displays tabular data as a table
+* :doc:`/components/console/helpers/progresshelper`: (deprecated) shows a progress bar
+* :doc:`/components/console/helpers/tablehelper`: (deprecated) displays tabular data as a table
 * :doc:`/components/console/helpers/dialoghelper`: (deprecated) interactively ask the user for information
```

Comment:
I'm actually not convinced by this change. What do you think if we removed the links to the deprecated helpers here and only talk about the ones introduced in Symfony 2.5? The deprecated helpers can still be accessed, but won't be advertised here.